lighthouse-core/audits/cache-start-url.js

```diff
@@ -39,20 +39,23 @@ class CacheStartUrl extends Audit {
    */
   static audit(artifacts) {
     let cacheHasStartUrl = false;
+    const manifest = artifacts.Manifest && artifacts.Manifest.value;
+    const cacheContents = artifacts.CacheContents;
+    const baseURL = artifacts.URL;
 
-    if (!(artifacts.Manifest &&
-          artifacts.Manifest.value &&
-          artifacts.Manifest.value.start_url &&
-          Array.isArray(artifacts.CacheContents) &&
-          artifacts.URL)) {
+    if (!(manifest && manifest.start_url && manifest.start_url.raw)) {
```

Comment:
`manifest.start_url` is always defined. It's [`manifest.start_url.raw`](https://github.com/GoogleChrome/lighthouse/blob/6663f6b0bab11f0971fe390ddbb2a167496d9c57/lighthouse-core/audits/cache-start-url.js#L58) that was causing the error before